examples/server_async.py

```diff
@@ -72,7 +72,7 @@ def setup_server(description=None, context=None, cmdline=None):
         # This is because many devices exhibit this kind of behavior (but not all)
         if args.store == "sequential":
             # Continuing, use a sequential block without gaps.
-            datablock = ModbusSequentialDataBlock(0x00, [1, 0 , 0] * 100)
+            datablock = ModbusSequentialDataBlock(0x00, [17] * 100)
```

Comment:
Why this ? I am pretty sure you cannot change this without making changes in the test suite as well.

Reply:
hmmm seems I was too quick. This change makes sense.